.gitignore

```diff
@@ -12,6 +12,7 @@
 /build
 
 # misc
+.env
```

Comment:
- [ ] you have put the `.env` file into the `.gitignore` file but I can still see that file into Github. Please remove that file because many times `.env` file contains the `API key` or any other sensitive information which you don't want to share with other users.

Reply:
Thanks for the review. I have removed it.